retis-events/src/display.rs

```diff
@@ -1,12 +1,24 @@
 use std::fmt;
 
 #[derive(Debug, Default, Clone, Copy, Eq, PartialEq)]
-pub enum DisplayFormat {
+pub enum DisplayFormatFlavor {
     SingleLine,
     #[default]
     MultiLine,
 }
 
+#[derive(Debug, Default, Clone, Copy)]
+pub struct DisplayFormat {
+    pub flavor: DisplayFormatFlavor,
```

Comment:
I was wondering whether or not to remove `DisplayFormatFlavor` entirely. I still don't have a strong opinion on this. The reasoning was this maps well to the cli `--format` option and is a top level configuration of the `DisplayFormat`, other embedded flags are more fine grained. In case we have more top-level configuration in the future, having an helper returning a `bool` wouldn't be easy to consume too. Or to say it differently, `DisplayFormatFlavor` would apply to everything in an event while embedded flags/values in `DisplayFormat` ar only for some parts.

But as said, no strong opinion.

Reply:
I guess that distinction does exist (whether it applies to the whole event or not) but as soon as some other option that applies to the full event is added, if it's orthogonal to multi/single line it will feel strange. E.g: `color: bool`.

Reply:
You had a point, makes sense.